core/rawdb/schema.go

```diff
@@ -93,6 +94,7 @@ type TxLookupEntry struct {
 	BlockHash  common.Hash
 	BlockIndex uint64
 	Index      uint64
+	SubIndex   uint64 // used for lookup CXReceipt from CXReceiptsProof
```

Comment:
We shouldn't add additional field. The old Index alone is enough to pinpoint the position of the receipt in the 2d array.

Reply:
we cannot use 2d array here. since each CXReceipts (array of CXReceipt) might have different length. so we need use 2d array to find location.

Reply:
What I mean is the index can be for the flattened receipts . Doesn’t matter what different length

Reply:
Okay, let use one index to save some space.